tests/integration/cattletest/core/test_host_stats.py

```diff
@@ -54,8 +54,8 @@ def test_stats_container(admin_client, docker_context):
     stats_access = container.stats()
 
     assert stats_access.token.index('.') > 0
```

Comment:
@alena1108 Yes, externalId now gets set for all containers via this line:
https://github.com/rancherio/python-agent/blob/1b87b33d26dcb0bc875b4982a2de29e0bc1deb54/cattle/plugins/docker/compute.py#L508.